meta-mender-demo/recipes-mender/boot-script/boot-script.bb

```diff
@@ -0,0 +1,19 @@
+LICENSE = "Apache-2.0"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+SRC_URI = "file://boot-script.service file://boot-script.sh file://LICENSE"
+
```

Comment:
Try to insert

```
S = "${WORKDIR}"
```

I think that will get rid of the `LICENSE` error you're getting in the Jenkins build.

Reply:
The secret way to get Jenkins to be happy? I like it ;)

Reply:
Hehe, yes, it's a long and complicated reason, so you just have to trust me that it *does* make sense! :-)